awx/ui_next/src/screens/Template/shared/SurveyToolbar.test.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { mountWithContexts } from '@testUtils/enzymeHelpers';
+import { act } from 'react-dom/test-utils';
+import SurveyToolbar from './SurveyToolbar';
+
+jest.mock('@api/models/JobTemplates');
+
+describe('<SurveyToolbar />', () => {
+  test('delete Button is disabled', async () => {
+    let wrapper;
+
+    await act(async () => {
+      wrapper = await mountWithContexts(
+        <SurveyToolbar
+          isDeleteDisabled
+          onSelectAll={jest.fn()}
+          isAllSelected
+          onToggleDeleteModal={jest.fn()}
+          onToggleSurvey={jest.fn()}
+        />
+      );
+    });
+
+    expect(wrapper.find('Button[variant="danger"]').prop('isDisabled')).toBe(
+      true
+    );
+  });
+
+  test('delete Button is enabled', async () => {
+    let wrapper;
+    await act(async () => {
+      wrapper = await mountWithContexts(
```

Comment:
- `mountWithContexts` [calls enzyme's mount function](https://github.com/ansible/awx/blob/devel/awx/ui_next/testUtils/enzymeHelpers.jsx#L136).

- [mount returns a ReactWrapper instance](https://github.com/enzymejs/enzyme/blob/master/docs/api/mount.md#returns), not a promise object

With these facts in mind, I don't think can use `await` on it. 

As an experiment, I tried changing all of the `wrapper = await mountWithContexts(` lines to `wrapper = mountWithContext` and the tests in this file all seem to run fine. Unless you need these for some reason, my recommendation would be to remove the`await` from the `mountWithContexts(` in all of your tests.